rosdep/base.yaml

```diff
@@ -59,6 +59,9 @@ apache2-mpm-prefork:
     wheezy: [apache2-mpm-prefork]
   gentoo: ['www-servers/apache[apache2_mpms_prefork]']
   ubuntu: [apache2-mpm-prefork]
+apparmor:
+  debian: [apparmor]
+  ubuntu: [apparmor]
```

Comment:
As we discussed above, this won't realistically ever be available for Fedora.  I'm thinking we should put an explicit `fedora: null` here, to indicate that it can't be available for Fedora (rather than just not being done, or something that could be done later).

Reply:
I never realized rosdistro was using `null` to explicitly state unavailability. If this is common practice, then I can update the PR accordingly.

Reply:
It seems to be a relatively new practice, and to be honest, the tools don't support it all that well (rosdep just throws a stacktrace if you try to install a null key).  But I think it should be straightforward enough to enhance the tools to make it nicer.

Reply:
Ok, I've added 61989c3